examples/crafted-early-init-elpaca.el

```diff
@@ -0,0 +1,83 @@
+;;; crafted-early-init-elpaca.el --- Bootstrap elpaca  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2023
+;; SPDX-License-Identifier: MIT
+
+;; Author: System Crafters Community
+
+;;; Commentary:
+
+;; Code to bootstrap elpaca package manager
+;;
+;; Load it in early-init.el like this:
+
+;; (setq crafted-emacs-home "~/crafted-emacs")
+;; (load (expand-file-name "custom-modules/crafted-early-init-elpaca"
+;;                         user-emacs-directory))
+
+;; on init.el you need to use crafted-package-install-selected-package 
+;; instead of package-install-selected-packages
+;;
+;; Packages phase
+;; ....
+;; (crafted-package-install-selected-packages)
+;; (alpaca-wait)
```

Comment:
This should be `(elpaca-wait)`, right?

Reply:
yes I confusing myself with the animal name jajaja